### PR TITLE
Amount formatter

### LINKED ISSUE #1313: [ADD] Comma seperators for large numbers on send and verify view
To have a better visibility for large numbers and amounts please add komma seperators for every `1000` .
Add this on `send view` , `verify view` and `done views`

### LINKED ISSUE #1369: [BUG] Copy amount with common in it result in two decimal points
**Describe the bug**

**To Reproduce**
1. Copy this number "11,111.11".
2. And paste it in the amount field.
3. It converts it to "11.111.11".

**Solution**
Update the formatter logic.

**Screenshots**
image

### PR DESCRIPTION
**Pull request by Rockindash**
PR fix for the following issues:
Comma separators for large numbers on send and verify view, Fixes #1313
Copy amount with common in it results in two decimal points, Fixes #1369

### CODE REVIEW

File: VultisigApp/VultisigApp/Views/Send/SendCryptoVerifyView.swift [hunk 1]
```diff
@@ -210,11 +210,11 @@ struct SendCryptoVerifyView: View {
     }
     
     private func getAmount() -> String {
-        tx.amount + " " + tx.coin.ticker
+        tx.amount.formatCurrencyWithSeparators(settingsViewModel.selectedCurrency) + " " + tx.coin.ticker
     }
     
     private func getFiatAmount() -> String {
-        tx.amountInFiat.formatToFiat()
+        tx.amountInFiat.formatToFiat().formatCurrencyWithSeparators(settingsViewModel.selectedCurrency)
```
Comment (johnnyluo) [inline VultisigApp/VultisigApp/Views/Send/SendCryptoVerifyView.swift:217]: image
Refer to here , the FiatAmount is empty, I am using USD as my currency

**Review by johnnyluo — CHANGES_REQUESTED** [review]
See comments @Rockindash

File: VultisigApp/VultisigApp/Views/Send/SendCryptoVerifyView.swift (hunk 1 above)
Comment (Rockindash) [reply VultisigApp/VultisigApp/Views/Send/SendCryptoVerifyView.swift:217]: Fixed the formatter's fallback value.
Simulator Screenshot - iPhone 15 Pro - 2024-11-06 at 18 08 17

**Review by johnnyluo — APPROVED** [review]
LGTM

**Pull request merged**